routes/api/orderAPI.js

```diff
@@ -0,0 +1,95 @@
+// Gravity Application Server | orderAPI
+// =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+module.exports = function (express) {
+  var router = express.Router();
```

Comment:
Per the airbnb style guide, all instances of 'var' should be changed to 'const'.